lib/KCard/index.vue

```diff
@@ -42,22 +45,25 @@
         </span>
       </div>
       <div
+        v-if="$slots.aboveTitle || preserveAboveTitle"
         data-test="aboveTitle"
         class="above-title"
       >
         <!-- @slot Places content to the area above the title. -->
         <slot name="aboveTitle"></slot>
       </div>
       <div
+        v-if="$slots.belowTitle || preserveBelowTitle"
         data-test="belowTitle"
         class="below-title"
       >
         <!-- @slot Places content to the area below the title. -->
         <slot name="belowTitle"></slot>
       </div>
       <div
+        v-if="$slots.footer || preserveFooter"
         data-test="footer"
-        class="footer"
+        :class="['footer', { 'footer--empty': preserveFooter && !$slots.footer }]"
```

Comment:
```suggestion
        class="footer"
```

**[required] Explained in the other comment.**